exchange/exchange_test.go

```diff
@@ -983,9 +984,10 @@ func TestCategoryMappingNoIncludeBrandCategory(t *testing.T) {
 
 	adapterBids[bidderName1] = &seatBid
 
-	bidCategory, adapterBids, err := applyCategoryMapping(nil, requestExt, adapterBids, categoriesFetcher, targData)
+	bidCategory, adapterBids, rejections, err := applyCategoryMapping(nil, requestExt, adapterBids, categoriesFetcher, targData)
 
 	assert.Equal(t, nil, err, "Category mapping error should be empty")
+	assert.Equal(t, 0, len(rejections), "There should be no bid rejection messages")
```

Comment:
Consider using `assert.Empty` instead.

Reply:
Updated to use `assert.Empty`, both here and other related tests.